src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -629,7 +648,7 @@ export default class FilingHistoryList extends Mixins(
         //   params: { filingId: filing.filingId, isCorrection: true } })
         // FOR NOW:
         this.$router.push({
-          name: Routes.CORRECTION,
+          name: routeName,
```

Comment:
At the moment, Annual Report corrections should continue to be handled using the original route. (It is a local filing, not a redirect to Edit UI.)

Ditto for COD and COA corrections below.